docs/modules/ROOT/pages/reference/cli.adoc

```diff
@@ -152,3 +152,17 @@ This command doesn't have any command line options.
 
 *--help*::
   Show component new usage and options then exit.
+
+== Inventory Components
+
+*-d, --distribution*::
```

Comment:
I suggest to go with a more generic approach. Can we add parameters that can be given more than once? If yes, can we have just a parameter to provide facts and one to provide parameters. With this, we then could also remove setting defaults for `parameters.cloud.availabilityZones` and `parameters.cloud.projectName`. Yes I am aware that this would make usage for as a bit more involved (adding them with each and every call) but it would keep this code a bit more clutter free. Alternatively, allow to pass them as a path to a yaml file.

Reply:
I've implemented the variant which allows passing an arbitrary number of extra YAML files which are used as additional inventory classes.